services/director-v2/src/simcore_service_director_v2/api/routes/clusters.py

```diff
@@ -195,3 +195,20 @@ async def test_cluster_connection(
         raise HTTPException(
             status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail=f"{e}"
         ) from e
+
+
+@router.post(
+    "/{cluster_id}:ping",
+    summary="Test cluster connection",
+    response_model=None,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def test_specific_cluster_connection(
+    user_id: UserID,
+    cluster_id: ClusterID,
+    clusters_repo: ClustersRepository = Depends(get_repository(ClustersRepository)),
+):
+    cluster = await clusters_repo.get_cluster(user_id, cluster_id)
+    return await test_gateway_endpoint(
```

Comment:
I see that the ping timeout is hard-coded. Don't  you want to make it a env var ?? Might be handy to adjust in ops depending how far the deployment is away from the clusters.

Reply:
hmm, yes and no. since the clusters may be anywhere... I will put an env but I guess this will be a bit of a difficult one.